package.json

```diff
@@ -88,6 +88,7 @@
     "babelrc-rollup": "^3.0.0",
     "chromatic": "^5.2.0",
     "codecov": "^3.8.0",
+    "colorette": "^1.2.0",
```

Comment:
We are using a matrixed build for node v12,13,14.x and Colorette drops support for Node < 14